targets/ipq806x

```diff
@@ -1,7 +1,29 @@
-ATH10K_PACKAGES='-kmod-ath10k kmod-ath10k-ct -ath10k-firmware-qca99x0 ath10k-firmware-qca99x0-ct'
+# The QCA9980 is discontinued from Qualcomm. It didn't receive any firmware-update since >4 years.
+# Any kind of meshing was never implemented for this chip. It is (most likely) broken forever.
```

Comment:
What does "Any kind of meshing" represent? Only 11s or also IBSS? Be explicit here. 
Also no need to put "most likely" in brackets.

Reply:
I do net own such a device, so i can't test them. The Firmware definitely does not support 11s, however it might support IBSS but as a single VAP (no additional client VAP). I can try to explain the situation more, but being 100% explicit is not possible for me here.